### PR TITLE
adding tfa2017 protocol (maybe identical to F007TH)

### PR DESCRIPTION
**Pull request by behrisch**
Apparently there is a newer protocol version used by the TFA Dostmann weather stations. It is very similar if not identical to the F007TH but I could not test this so I went with the TFA name.

### CODE REVIEW

File: libs/pilight/protocols/433.92/tfa2017.c [hunk 1]
```diff
@@ -208,7 +211,7 @@ __attribute__((weak))
 void tfa2017Init(void) {
 	protocol_register(&tfa2017);
 	protocol_set_id(tfa2017, "tfa2017");
-	protocol_device_add(tfa2017, "tfa2017", "TFA 30.X Temp Hum Sensor Protocol 2017");
+	protocol_device_add(tfa2017, "tfa2017", "TFA 30.X Temp Hum Sensor Revision 08/2017");
```
Comment (CurlyMoo) [inline libs/pilight/protocols/433.92/tfa2017.c]: Better

**Comment by CurlyMoo** [comment]
Is this ready as well?

**Comment by behrisch** [comment]
Yes.

**Comment by CurlyMoo** [comment]
Can you change the license here as well?

File: docs/source/protocols/433.92/weather/tfa2017.rst [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+| humidity-decimals    | 2           | number     | How many decimals the GUIs should display for humidity    |
++----------------------+-------------+------------+-----------------------------------------------------------+
+| show-temperature     | 1           | 1 or 0     | Don't display the temperature value                       |
++----------------------+-------------+------------+-----------------------------------------------------------+
+| show-humidity        | 1           | 1 or 0     | Don't display the humidity value                          |
++----------------------+-------------+------------+-----------------------------------------------------------+
+
+.. rubric:: Protocol
+
+The protocol is similar to the F007TH as described in `this document <http://www.osengr.org/WxShield/Downloads/Weather-Sensor-RF-Protocols.pdf>` starting at page 13.
```
Comment (CurlyMoo) [inline docs/source/protocols/433.92/weather/tfa2017.rst]: This link isn't parsed correctly. It should be like this:
```rst
in `this <http://www.osengr.org/WxShield/Downloads/Weather-Sensor-RF-Protocols.pdf>`_ document
```

**Comment by CurlyMoo** [comment]
I'll ask again because of the latest changes in the manual. Ready to merge?

**Comment by behrisch** [comment]
Yes, I just took the chance to clarify a few things and fix the revision date for the protocol. I think it is still wrong in the rewrite branch.

**Comment by CurlyMoo** [comment]
> I think it is still wrong in the rewrite branch.

Let's leave that for what it is.

**Pull request merged**